db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/App.kt

```diff
@@ -20,19 +21,27 @@ fun main() {
 fun RapidsConnection.createDb(database: Database, repository: Repository): RapidsConnection {
     sikkerLogger.info("Starter Flyway migrering...")
```

Comment:
fjerne denne.. Generelt kan vel alt i denne klassen logge til vanlig logg, trenger ikke sikker?

Reply:
Mitt syn er at så mye som mulig i appens levetid (start, stopp, eventer og feil) bør logges til åpen logg, da ser man også at app lever og logger det som skjer, uten at man trenger å oppsøke sikre logger. Sikkerlogg bør egentlig ikke være "nødvendig", bortsett fra at det kan være praktisk for å inspisere faktiske data som fremprovoserte en feil etc, så jeg ser på sikkerlogg litt mer som en debug-logg